cypress/steps/salesSteps.js

```diff
@@ -0,0 +1,59 @@
+import moment from "moment-timezone";
+
+import { ASSIGN_PRODUCTS_SELECTORS } from "../elements/catalog/assign-products";
+import { MENAGE_CHANNEL_AVAILABILITY } from "../elements/channels/menage-channel-availability";
+import { SALES_SELECTORS } from "../elements/discounts/sales";
+import { BUTTON_SELECTORS } from "../elements/shared/button-selectors";
+
+class SalesSteps {
+  discountOptions = {
+    PERCENTAGE: SALES_SELECTORS.percentageOption,
+    FIXED: SALES_SELECTORS.fixedOption
+  };
+
+  createSale({
+    saleName,
+    channelName,
+    discountValue = 10,
+    discountOption = this.discountOptions.PERCENTAGE
+  }) {
+    cy.get(SALES_SELECTORS.createSaleButton)
+      .click()
+      .get(SALES_SELECTORS.nameInput)
+      .type(saleName)
+      .get(discountOption)
+      .click()
+      .get(MENAGE_CHANNEL_AVAILABILITY.availableManageButton)
+      .click()
+      .get(MENAGE_CHANNEL_AVAILABILITY.allChannelsInput)
+      .click()
+      .get(MENAGE_CHANNEL_AVAILABILITY.channelsAvailabilityForm)
+      .contains(channelName)
+      .click()
+      .get(BUTTON_SELECTORS.submit)
+      .click()
+      .get(SALES_SELECTORS.discountValue)
+      .type(discountValue)
+      .get(SALES_SELECTORS.startDateInput)
+      .type(moment().format("YYYY-MM-DD"));
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat#Syntax
Wont be sufficient? Moment is a big pkg, so I'm against adding it to thee project just for one date format in tests.